Shared/Samples/Navigate route/NavigateRouteView.swift

```diff
@@ -154,7 +154,7 @@ private extension NavigateRouteView {
             autoPanMode = .off
             
             // Creates the graphics for each stop.
-            let stopGraphics = stops.map {
+            let stopGraphics = Self.stops.map {
```

Comment:
Was this causing an issue with the preview for you? It seems to work okay for me without these changes.

Reply:
Yes, I get `Compiling failed: cannot find 'stops' in scope`. The previews have had a hard time recognizing `static` variables and nested types for me. The alternative here is to use `NavigateRouteView.stops`, but it made more sense to me to put it on the model since it is only used by the model.